### ISSUE TITLE
SHFB: Error BE0026: Unable to determine default topic in toc.xml.  

### ISSUE THREAD

**Issue by mjeaton** [EWSoftware/SHFB#14]
I grabbed the latest code a couple days ago, built it and am trying to build our help. I opened our older project file (built using 1.9.1.0), made a few changes (copyright, etc.), but now I'm struggling to get the project built.

Our project has a Content Layout.content file that points to several .aml files. Initially, I was getting BE0026, so I followed the help and made sure one of the topics was marked as the default, but I'm still seeing BE0026 when I try to build.

I'm not sure what else to do.

Thoughts?

**Comment by EWSoftware** [comment]
I don't know of any reasons it shouldn't just work.  Can I download it from somewhere or can you e-mail me the project or an example that duplicates the issue?

**Comment by mjeaton** [comment]
I'll see what I can do! Thanks Eric!

**Comment by EWSoftware** [comment]
Were you able to resolve this?

**Comment by mjeaton** [comment]
Actually, not yet. Priorities shifted, but I'll carve out some time today or tomorrow to see if I can get a reproducible project.

**Comment by rdepena** [comment]
Just installed version 15.1.12.0 and I got this error, my co-worker has version 2014.11.22.0 and is able to build on the same project.

**Comment by EWSoftware** [comment]
@rdepena  I'm still waiting on an example that I can use to reproduce the problem.  If you can send me one, I'll look into it.

**Comment by EWSoftware** [comment]
No further response so I'm closing this for now.  If an example can be supplied, feel free to re-open the issue.

**Comment by EWSoftware** [comment]
I've been making some updates to the build engine and took another look into this.  I was able to generate the error but only after setting up a rather convoluted scenario with a content layout file containing an empty place holder root topic with a child topic that was hidden in the TOC.

Anyway, in reviewing the code I did notice some inefficiencies with the way the default topic was being determined so I reworked the code in question and unless something really odd is encountered there's little if any chance that this error will occur in future releases.
